### DISCUSSION TITLE
Figma link?

### DISCUSSION THREAD

**Discussion by ducesane** [dugsiiyeinc/dhaqsoPay#33] [dhaqsoPay Extension Developers]
xage laga helaa figma  Design xalay Mohamed soo bandhigay , Mideeda kale  pages-ka extensions-ka aan qeybsano si aaan shaqada u galno

**Comment by iftiiin** [comment]
Mohamed aa haayo asagaa nala soo wadaagi doono i.a.

**Reply by ducesane** [reply]
insh allah  waa sugeynaa

**Comment by hayle102** [comment]
linkiga waxaad ka heleysaan
#42
